srcpkgs/xbps-triggers/files/dkms

```diff
@@ -70,9 +70,9 @@ add_modules() {
 			echo "Skipping kernel-${_kver}. kernel-headers package not installed..."
 			continue
 		fi
-		if [ ! -f ${f}/build/scripts/basic/fixdep ] ; then
-			echo -n "Building scripts for kernel-${_kver}... "
-			yes "" | make -j$(nproc) -C ${f}/build scripts > ${f}/build/make.log 2>&1
+		if [ ! -f ${f}/build/scripts/basic/fixdep ] || [ ! -f ${f}/build/scripts/mod/modpost ] ; then
+			echo -n "Prepare to build modules for kernel-${_kver}... "
+			yes "" | make -j$(nproc) -C ${f}/build prepare0 > ${f}/build/make.log 2>&1
```

Comment:
How does this fail if someone doesn't have gcc installed?

Reply:
I don't get what you mean, I'm afraid. dkms depends on gcc

Reply:
Oops, sorry. For some reason I was thinking this would be run when someone installed the headers package.

I haven't tested, though.